vkcs/provider_test.go

```diff
@@ -320,3 +325,42 @@ func testAccRenderConfig(testConfig string, values ...map[string]string) string
 
 	return buf.String()
 }
+
+func getStepsWithMigrationCases(steps []resource.TestStep) (migrationSteps []resource.TestStep) {
+	for _, step := range steps {
+		migrationSteps = append(migrationSteps, resource.TestStep{
+			Config: step.Config,
+			Check:  step.Check,
+			ExternalProviders: map[string]resource.ExternalProvider{
+				"vkcs": {
+					VersionConstraint: ">= 0.2.1",
```

Comment:
I think the point of this test is to test migration from version where resource was written with SDKv2 to current version where we use Framework to write it. So, the version constraint will cause the test to do nothing useful, when resource will be written with Framework in both previous and current versions. Thus, I would suggest to pin version to 0.2.1 for datastore capabilities.

Reply:
You're right, pinned the version